src/llvm_backend_proc.cpp

```diff
@@ -2174,7 +2174,32 @@ gb_internal lbValue lb_build_builtin_proc(lbProcedure *p, Ast *expr, TypeAndValu
 			case 128: return lb_emit_runtime_call(p, "abs_complex128", args);
 			}
 			GB_PANIC("Unknown complex type");
+		} else if (is_type_float(t)) {
```

Comment:
Is there an LLVM intrinsic we could use rather than doing this manually?

Reply:
There is (https://llvm.org/docs/LangRef.html#llvm-fabs-intrinsic), I thought you didn't use it initially for a good reason, I can use that instead.

Reply:
I cannot remember why I didn't use it. Maybe it required linking with libm?

Reply:
I've tried it and it does not require libm, it does generate the exact same code as what I do here. But for the endian types we would need to byteswap, call intrinsic, and byteswap back, which may be worse.

What do you want me to do, call the intrinsic, or keep it like this?

Reply:
Keep it like this, I guess.